react-app/src/ExerciseOverview/ExerciseDetail.js

```diff
@@ -98,7 +99,7 @@ const styles = StyleSheet.create({
     color: ThemeColor.textColor,
   },
   instructionContainer: {
-    height: 230,
+    flex: 1,
```

Comment:
I think `flex:1` is not enough to fix the bug. Please check the exercise detail in the first move, still can't scroll down to the bottom. 
I guess the key is in the `container`, `top: 150,` and `height: '100%',` it push the whole page to lower of the screen.